app/api/datastore/base.rb

```diff
@@ -23,16 +23,16 @@ class Base < Grape::API
         optional(
           :page,
           type: Integer,
-          default: 1,
+          default: Pagination::DEFAULT_PAGE,
           desc: 'Page to fetch.'
         )
 
         optional(
           :per_page,
           type: Integer,
-          default: 20,
+          default: Pagination::DEFAULT_PER_PAGE,
           desc: 'Number of results to return per page.',
-          values: 1..200
+          values: 1..Pagination::MAX_PER_PAGE
```

Comment:
On Apply we've been thinking if allowing to pass 0 `per_page` might be a sensible approach for requests where we are only interested in the total number of records. That will essentially return to us the paginator object with the metadata, but an empty records collection.

Not a big prob, as right now we are using limit 1 which is still efficient.
This is not a blocker, just crossed my mind.

Reply:
Good point. We can always add them to the headers and do an headers only request.